app/views/course/assessment/assessments/requirements.json.jbuilder

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+requirements = @assessment.specific_conditions.filter_map do |condition|
```

Comment:
Move this logic to the controller. As much as possible, we should abstract logic out of view. It would be easier to debug as well since currently there is no `requirements` function in the controller.. One might think it's too magic that suddenly this json is called out of nowhere..